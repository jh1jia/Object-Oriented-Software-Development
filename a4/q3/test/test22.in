test22.txt dropfirst 6
