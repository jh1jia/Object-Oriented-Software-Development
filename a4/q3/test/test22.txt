DDDDdddd abcdefgH
D023aAc3 45%dC
c C cC Cc 1Dd20#dCe00
00
