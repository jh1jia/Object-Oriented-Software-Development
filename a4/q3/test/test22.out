dd
gH
c3
dCe00
